### DISCUSSION TITLE
Using ticks and TTL to sync

### DISCUSSION THREAD

**Discussion by benefron** [bonsai-rx/bonsai#1257] [Q&A]
Hi,
I wanted to know how reliable is the clock that Bonsai uses. I guess it using the computer clock.
If I feed Bonsai with a sync TTL and collect the data from timestamp ticks are the ticks the same across all timestamp nodes in Bonsai?
For example, if I timestamp every frame from a camera and timestamp every sync TTL is it on the same clock, an I assume that the times are shared?
Thank you

**Comment by bruno-f-cruz** [comment]
Hard to answer without knowing the exact hardware you are using (camera and ttl reader).  In general, from past benchmarks, you can get a jitter of around 10ms in optimal conditions (using harp hardware for instance), but >20ms is not uncommon.
